shm/src/Store.cpp

```diff
@@ -87,51 +87,83 @@ void Store::generateAllCargo() {
 }
 
 void Store::convertDataFromFile(std::string lineFromFile) {
-    std::vector<std::string>dataToCreateItems(8, "");
-    size_t iterator{};
+    std::string singleWordFromFile;
+    size_t wordNumber{};
     for (size_t i = 0; i < lineFromFile.size(); i++) {
         if (lineFromFile[i] != ' ') {
-            dataToCreateItems[iterator] += lineFromFile[i];
+            singleWordFromFile += lineFromFile[i];
         } else {
-            iterator++;
+            wordNumber++;
+            valueOneLineAssignment(wordNumber, singleWordFromFile);
         }
     }
-    generateSingleCargo(dataToCreateItems);
+    generateSingleCargo();
 }
 
-void Store::generateSingleCargo(std::vector<std::string>const & cargo) {
-    if (cargo[0] == "Fruit"){
-        generateFruit(cargo);
-    } else if (cargo[0] == "Alcohol"){
-        generateAlcohol(cargo);
-    } else if (cargo[0] == "Item"){
-        generateItem(cargo);
+void Store::valueOneLineAssignment(const size_t & wordNumber, const std::string & singleWordFromFile) {
+    switch (wordNumber) {
+        case 1 : {
+            oneLine_.className_ = singleWordFromFile;
+        } break;
```

Comment:
It could be better to change formatting:
```cpp
switch (wordNumber) {
case 1:
     oneLine_.className_ = singleWordFromFile; break;
case 2 : 
     oneLine_.itemName_ = singleWordFromFile; break;
case 3:
...
}
```
because otherewise switch becomes excessively long.